deposit/templates/deposit/data.html

```diff
@@ -133,7 +133,7 @@ <h5>Yoda iCommands</h5>
                     Therefore, it is possible to transfer data to and from Yoda using the iRODS communication protocol.
                     You need to configure the iCommands to connect to this Yoda environment.
                 </p>
-                <a href="#" title="iCommands configuration" class="btn btn-sm btn-secondary" data-bs-toggle="modal" data-bs-target="#aboutModal">iCommands Configuration</a>
+                <a href="https://portal.yoda.test/user/data_transfer" title="iCommands configuration" class="btn btn-sm btn-secondary">iCommands Configuration</a>
```

Comment:
In the templates we should use the url_for function to refer to other modules / templates.

Reply:
changed it! thanks